espressif/vscode-esp-idf-extension
# Bugfix/idf versions

## Description

- Add download with retries to idf_versions.txt file and update local fallback file.
- Add timestamps IDF version validation (IDF >= 4.4)
- Add changelog for v1.6.4 release
- Fix workspace folders scope in TaskManager.addTask
- Update readme

Fixes #998
Fixes #990 

## Type of change

Please delete options that are not relevant.
- Bug fix (non-breaking change which fixes an issue)

## Steps to test this pull request

Provide a list of steps to test changes in this PR and required output

1. Click on "Configure ESP-IDF extension" command
2. Execute action.
3. Observe results. The setup wizard should appear and work.

- Expected behaviour:

- Expected output:

## How has this been tested?

Manual testing

**Test Configuration**:
* ESP-IDF Version: 5.0
* OS (Windows,Linux and macOS): macOS

## Checklist
